app/services/lesson_duration.rb

```diff
@@ -29,6 +29,7 @@ def durations
   end
 
   def previous_lesson
-    FindLesson.new(lesson, lesson.course).previous_lesson
+    course = lesson.course
+    course.lessons.where('lessons.position < ?', lesson.position).last
```

Comment:
We don't use this rake task at all. I'll be removing this and it's associated services in a follow PR.